website/blog/2022-05-26-aluminium.md

```diff
@@ -0,0 +1,246 @@
+---
+author: Rikito Taniguchi
+title: Metals v0.11.6 - Aluminium
+authorURL: https://twitter.com/tanishiking25
+authorImageURL: https://github.com/tanishiking.png
+---
+
+We're happy to announce the release of Metals v0.11.6, which 
+
+<table>
+<tbody>
+  <tr>
+    <td>Commits since last release</td>
+    <td align="center">119</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">46</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">12</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center"></td>
+  </tr>
+  <tr>
+    <td>New features</td>
+    <td align="center"></td>
+  </tr>
+</tbody>
+</table>
+
+For full details: https://github.com/scalameta/metals/milestone/<num>?closed=1
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs and
+Sublime Text. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with the help from [Lunatech](https://lunatech.com) along with contributors from
+the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+
+## Reduce file watcher memory usage
+Previously, Metals uses `FileTreeRepository` of [swoval](https://github.com/swoval/swoval) for a file watcher, that maintains an in memory cache of some subset of the file system.
+Now, Metals uses `PathWatcher` of [swoval](https://github.com/swoval/swoval) instead. This implementation no longer caches the watched file trees, and Metals consumes smaller amount of memory.
+
+For more details, see the original PR: [\#3758](https://github.com/scalameta/metals/pull/3758)
```

Comment:
@pvid I wrote some words about https://github.com/scalameta/metals/pull/3758 but if you want to change something, please let me know :)

Reply:
You could maybe mention that this is meant to help with working on very large workspaces - I think that is a more "product oriented view" than describing the implementation details